libs/crosstarget-utils/src/common/regex.rs

```diff
@@ -0,0 +1,42 @@
+use std::fmt::Display;
+
+#[derive(Debug)]
+pub struct RegExpError {
+    pub message: String,
+}
+
+impl Display for RegExpError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "Regular expression error: {}", self.message)
+    }
+}
```

Comment:
Perfectly fine as it is, but fyi we recently started introducing the `derive_more` crate in the QE. It makes these things easier:
```suggestion
use derive_more::Display;

#[derive(Debug, Display)]
#[display("Regular expression error: {message}")]
pub struct RegExpError {
    pub message: String,
}
```

Up to you if you want to use it here.

Reply:
Thanks for raising that, I didn't know about it.